AmplifyPlugins/Auth/AWSCognitoAuthPlugin/Dependency/AuthorizationProviderAdapter+SignedInSession.swift

```diff
@@ -58,6 +58,10 @@ extension AuthorizationProviderAdapter {
                         self.fetchSignedInSession(withError: AuthErrorHelper.toAuthError(error!),
                                                   completionHandler)
                     }
+                } else if (error as NSError?)?.userInfo["__type"] as? String == "UserNotFoundException" {
```

Comment:
Can we map the NSError as done in the function `isErrorCausedByMisconfiguredIdentityPool`

Reply:
Addressed in [PR #1605](https://github.com/aws-amplify/amplify-ios/pull/1605)